core/code/entity_info.js

```diff
@@ -15,8 +16,10 @@ window.teamStringToId = function(teamStr) {
   var team = TEAM_NONE;
   if(teamStr === 'ENLIGHTENED') team = TEAM_ENL;
   if(teamStr === 'RESISTANCE') team = TEAM_RES;
+  if (teamStr === 'U̶͚̓̍N̴̖̈K̠͔̍͑̂͜N̞̥͋̀̉Ȯ̶̹͕̀W̶̢͚͑̚͝Ṉ̨̟̒̅') team = TEAM_MAC;
```

Comment:
I didn't see this string except for portal owner and layer display name, I'm expecting `"MACHINA"` instead